src/config/constants.ts

```diff
@@ -14,7 +14,8 @@ export const OPENSHIFT_VERSION_OPTIONS: OpenshiftVersionOptionType[] = [
 export const CLUSTER_MANAGER_SITE_LINK =
   'https://cloud.redhat.com/openshift/install/metal/user-provisioned';
 
-export const POLLING_INTERVAL = 10000;
+export const POLLING_INTERVAL = 10 * 1000;
+export const EVENTS_POLLING_INTERVAL = 60 * 1000;
```

Comment:
I am not sure about such long interval. I think 30 seconds would be reasonable delay. @jtomasek , wdyt?

Reply:
IMO the cluster polling interval should be kept at 10s, we need to keep the UI up to date - state changes etc.
60s for events is probably fine

Reply:
Ok, changing the cluster to 10 secs to keep the rendered info up-to-date.

I still think we should decrease the time for event polling at least once we move the events to a modal.